pkg/prove/proof.go

```diff
@@ -22,99 +22,52 @@ import (
 	"github.com/tendermint/tendermint/types"
 )
 
-// TxInclusion uses the provided block data to progressively generate rows
-// of a data square, and then using those shares to creates nmt inclusion proofs.
-// It is possible that a transaction spans more than one row. In that case, we
-// have to return more than one proof.
```

Comment:
Is this no longer possible?

Reply:
I think it is still possible. I found the godoc confusing because only one `types.TxProof` is returned but this godoc is alluding to the fact that more than one proof may be present at `types.TxProof.Proofs`. I'll make an attempt at clarifying the godoc after we resolve https://github.com/celestiaorg/celestia-core/issues/948

Reply:
No longer applicable b/c this function returns a singular share proof.